odoo/addons/base/data/res.country.state.csv

```diff
@@ -297,8 +297,8 @@ state_eg_sin,eg,"North Sinai","SIN"
 state_eg_shg,eg,"Sohag","SHG"
 state_za_ec,za,"Eastern Cape","EC"
 state_za_fs,za,"Free State","FS"
-state_za_gt,za,"Gauteng","GT"
-state_za_nl,za,"KwaZulu-Natal","NL"
+state_za_gt,za,"Gauteng",GP
+state_za_nl,za,"KwaZulu-Natal",KZN
```

Comment:
Indeed, thanks for the change :+1: but it looks like you forgot the quotation mark
```suggestion
state_za_gt,za,"Gauteng","GP"
state_za_nl,za,"KwaZulu-Natal","KZN"
```